tests/test_vnet.py

```diff
@@ -894,12 +900,16 @@ def check_vnet_routes(self, dvs, name, endpoint, tunnel, mac="", vni=0):
         _vni = str(vni) if vni != 0 else self.vnet_map[name]['vni']
 
         if (mac,_vni) not in self.vnet_mac_vni_list:
-            new_fdb = get_created_entry(asic_db, self.ASIC_FDB_ENTRY, self.fdbs)
+            new_fdbs = get_all_created_entries(asic_db, self.ASIC_FDB_ENTRY, self.fdbs)
```

Comment:
do you know why there're extra FDB entries in the ASIC database?

Reply:
No, I tried many times and  I wasn't able to reproduce it in my environment. Probably it's specific to public environment were VS tests are running.

Reply:
ok. thanks.